src/app/pages/Home/Home.tsx

```diff
@@ -1,20 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Header from '../../components/Header/Header';
 import Searchbar from '../../components/Searchbar/Searchbar';
 import Navigation from '../../components/navigation/Navigation';
 import styles from './Home.module.css';
+import type { Doctor } from '../../lib/types';
+import DoctorCard from '../../components/DoctorCard/DoctorCard';
 
 export default function Home(): JSX.Element {
+  const [searchValue, setSearchValue] = useState('');
+  const [doctors, setDoctors] = useState<Doctor[]>([]);
```

Comment:
This is how `useLocalStorage` would work
```suggestion
  const [doctors, setDoctors] = useLocalStorage<Doctor[]>(mockedDoctors);
```
However, then you would need to use a different state for rendering and filtering your doctors. In that case you won't need another custom hook